egl/egl_util.c

```diff
@@ -97,3 +98,4 @@ EGLImageKHR createEglImageFromHandle(EGLDisplay eglDisplay, EGLContext eglContex
 
     return eglImage;
 }
+#endif
```

Comment:
In principle, we should not have a ENABLE_EGL within a EGL file.  If we're compiling this on non-egl scenarios that has to be fixed.